index.js

```diff
@@ -676,16 +691,24 @@ class GitlabScm extends Scm {
      * @return {Promise}
      */
     _getBellConfiguration() {
-        const bellConfig = {
+        const scmContexts = this._getScmContexts();
+        const scmContext = scmContexts[0];
+        const cookie = this.config.gitlabHost
+            ? `gitlab-${this.config.gitlabHost}`
+            : 'gitlab-gitlab.com';
```

Comment:
you can reduce use of a hardcode by defining the default host to an instance variable.